tests/test_scanimage_extractor.py

```diff
@@ -1026,60 +1082,473 @@ def test_plane_index_with_multiple_frames_per_slice(self):
             )
 
 
-def test_get_frames_per_slice():
```

Comment:
the diff algorithm failed on this one. Those tests were not removed, they are at the bottom.